lang/fr.json

```diff
@@ -67,25 +67,25 @@
 "Dump":"Extraction",
 "Dumping options.":"Options d'extraction.",
 "Dump NAND image, separate partitions or files from the SysNAND.":"Extraire l'image de la NAND, des partitions séparées ou des fichiers à partir de la SysNAND.",
-"Dump NAND image, separate partitions or files from the first EmuNAND.":"",
-"Dump NAND image, separate partitions or files from the second EmuNAND.":"",
-"Dump NAND image, separate partitions or files from the third EmuNAND.":"",
-"Image":"Образ",
+"Dump NAND image, separate partitions or files from the first EmuNAND.":"Extraire l'image de la NAND, des partitions séparées ou des fichiers à partir de la première EmuNAND.",
+"Dump NAND image, separate partitions or files from the second EmuNAND.":"Extraire l'image de la NAND, des partitions séparées ou des fichiers à partir de la seconde EmuNAND.",
+"Dump NAND image, separate partitions or files from the third EmuNAND.":"Extraire l'image de la NAND, des partitions séparées ou des fichiers à partir de la troisième EmuNAND.",
+"Image":"Image",
 "Backup NAND raw image to file.":"",
-"Partitions":"",
-"Dump decrypted NAND partitions to file.":"",
-"Dump decrypted TWL (DSi) partition to file.":"",
-"Dump decrypted TWL (DSi) FAT16 partition (TWL logical partition 0) to file.":"",
-"Dump decrypted TWL (DSi) FAT12 photo partition (TWL logical partition 1) to file.":"",
-"Dump decrypted AGB (GBA) savegame partition to file.":"",
+"Partitions":"Partitions",
```

Comment:
If string is exactly the same as in english varient, it should not be placed inside of the translation.